CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- `Finally` control flow graph blocks to be `Terminated` with a `FinallyBlockNode`.
```

Comment:
These are specific implementation details for a (currently) non-public API.
What problem does it solve, or how does the change actually surface to the analysis?